lib/finders/worldwide_organisations.json

```diff
@@ -0,0 +1,30 @@
+{
+  "base_path": "/government/world/organisations",
+  "title": "Worldwide organisations",
+  "description": "",
+  "locale": "en",
+  "document_type": "finder",
+  "schema_name": "finder",
+  "publishing_app": "whitehall",
+  "rendering_app": "finder-frontend",
+  "details": {
+    "document_noun": "organisation",
+    "default_order": "title",
+    "facets": [],
+    "filter": {
+      "format": "worldwide_organisation"
+    },
+    "format_name": "Worldwide organisations",
+    "show_summaries": true
```

Comment:
You are correct. This would show a small text under the title, but there isn't one here.